gallery/gallery-library/src/integration-test/java/org/silverpeas/components/gallery/GalleryWarBuilder.java

```diff
@@ -23,6 +23,7 @@
  */
 package org.silverpeas.components.gallery;
 
+import org.silverpeas.components.gallery.stub.SessionManagementStub;
```

Comment:
Why not use the one implemented into core-web-test?

Reply:
See my answer above

Reply:
:thumbsup: